test/FakeRouteCollector.php

```diff
@@ -0,0 +1,10 @@
+<?php
+declare(strict_types=1);
+
+namespace FastRoute\Test;
+
+use FastRoute\RouteCollector;
+
+class FakeRouteCollector extends RouteCollector
```

Comment:
This is not needed, as we just need to implement the interface.

Reply:
/me nods along

That would mean (as I noted above) copying the entire RouteCollector class to replace it. That means, in turn, any changes to its behavior would mean copying those changes to the replacement class. That sounds like something to be avoided, if at all possible. Do you kind of see what I'm getting at?